README.md

```diff
@@ -22,6 +23,7 @@ What is the purpose of this file, and what is the purpose of the numbers at the
 
 ```text
 Your answer...
+This creates a migrations file database. The number presented in the beginning of the files name contains the date and time the file was created.
```

Comment:
:+1:
It is just a migrations file (not file database).  When run, it will change the schema of the database.  Please review our suggested answer (in the solution branch) for more info.